R/droplet-ssh.R

```diff
@@ -108,8 +108,14 @@ droplet_ip <- function(x) {
     stop("No network interface registered for this droplet\n  Try refreshing like: droplet(d$id)",
       call. = FALSE)
   }
-
-  v4[[1]]$ip_address
+  ips = do.call("rbind", lapply(v4, as.data.frame))
+  public_ip = ips$type == "public"
+  if (!any(public_ip)) {
+    ip =v4[[1]]$ip_address
+  } else {
+    ip = ips$ip_address[ public_ip] [[1]]
```

Comment:
seems like `[[1]]` is not needed here? maybe it is sometimes?  also, remove the extra spaces such as in `[ public_ip]`

Reply:
Some may have multiple IP - so just in case.

Reply:
Okay. Sounds good to leave `[[1]]`, but the extra space is still there.

Reply:
Not seeing it at https://github.com/sckott/analogsea/pull/198/files#diff-78595aaefd5e15ea3f9301b57f6106cdR118

Reply:
Oh you mean the space before `[[1]]`.  Hmm - Styler put the space in there I think - will remove.